docs/volto/bootstrap.md

```diff
@@ -46,7 +46,7 @@ yo @plone/volto
 ```
 
 Follow the prompts' questions, providing `my-volto-app` as the project name.
-
+**Note:** Do not start your project name with `volto`, as it will cause `plone/generator-volto` to fail
```

Comment:
We use MyST directives for [admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html). Also add a blank line before and after.

````suggestion

```{warning}
Do not start your project name with `volto`, as it will cause `plone/generator-volto` to fail.
```

````

Reply:
Thanks for letting me know this. I didn't knew the correct method, but thanks to you now I know it.

I m updating my PR as per your suggestions